deep_duplication.rb

```diff
@@ -0,0 +1,27 @@
+# Pass-by-reference is an important concept. Create a method deep_dup which returns a completely new object on a 2-or-more dimensional array. Ruby's simple dup method will be useful but not the only thing you use.
+def deep_dup(original)
+
+  # Using the Marshal class, which is normally used for serialization, you can create a ‘deep copy’ of an object.
+  duper = Marshal.load(Marshal.dump(original))
```

Comment:
This solution is correct. But you are expected to solve this using native array methods.